docs/dasharo-tools-suite/documentation.md

```diff
@@ -447,3 +448,53 @@ version. To properly update it, follow these steps:
   board: clevo/ns50mu
   version: 2022-08-31_cbff21b
   ```
+
+### Run commands from iPXE shell
```

Comment:
This for me looks like developer or OEM feature. How it is useful for regular users, which want to enable Dasharo just for one unit?

Diataxis is very important, we should try split documentation targeted for various audience.

Reply:
@pietrushnic could you look at this and tell if adding new issue can resolve this conversation?

Reply:
As mentioned above, we added issue to track this problem, documentation needs to be more user friendly, this is something that we would like to improve in future releases. I will resolve this conversation now